src/main/java/com/epam/grid/engine/service/JobOperationProviderService.java

```diff
@@ -57,9 +60,12 @@ public class JobOperationProviderService {
      * @see JobProvider
      */
 
-    public JobOperationProviderService(final JobProvider jobProvider, @Value("${job.log.dir}") final String logDir) {
+    public JobOperationProviderService(final JobProvider jobProvider,
+                                       @Value("${job.log.dir}") final String logDir,
+                                       @Value("${grid.engine.shared.folder}") final String gridSharedFolder) {
         this.jobProvider = jobProvider;
         this.logDir = logDir;
```

Comment:
here we also should construct it as for `SGE` or `SLURM` providers

Reply:
done